### DISCUSSION TITLE
Welcome to quantum-kernel-training Discussions!

### DISCUSSION THREAD

**Discussion by caleb-johnson** [qiskit-community/prototype-quantum-kernel-training#1] [Announcements]
We’re using Discussions as a place to connect with researchers, developers, and other members of our community. We hope that you:

Ask questions you’re wondering about.
Share ideas.
Engage with other community members.
To get started, comment below or create your own post with anything you would like to share.

For bug reports and feature requests, please use Github Issues.

**Comment by vignanreddy2907** [comment]
What is advantage of Quantum Machine Learning over Classical ML, is it accuracy or computation time.
i mean in what cases and scenarios(data sets) it is better use QML instead of Classical ML?
when i performed classification of iris data using SVC and QSVC, SVC gave more accuracy in less time compare to QSVC on ibmq_qasm_simulator.
